src/update-manager.js

```diff
@@ -349,13 +349,15 @@ class UpdateManager {
 
         console.log('UpdateManager: update Ok for <' + uri + '>')
 
-        callback(uri, response.ok, response.responseText, response)
+        callbackFunction(uri, response.ok, response.responseText, response)
       })
       .catch(err => {
-        callback(uri, false, err.message, err)
+        callbackFunction(uri, false, err.message, err)
```

Comment:
Might want to consider `(error, result)` for callbacks in the future, where `result` is an object that combines the multiple things.